shared/src/commonMain/kotlin/at/asitplus/wallet/app/common/Extensions.kt

```diff
@@ -109,9 +113,14 @@ fun DCQLCredentialQuery.extractConsentData(): Triple<CredentialRepresentation, C
         else -> PLAIN_JWT
     }
 
+    // TODO Show all attributes as requested, without matching them against our supported schemes
```

Comment:
If we have the function above, we might also be able to just list the requested claims over DCQL here, right? @acrusage-iaik ?

Reply:
Sounds fine as well, could you please sketch it out or build upon this PR?